Tests/BenchmarkTests/BenchmarkSuiteTests.swift

```diff
@@ -0,0 +1,22 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Benchmark
+import BenchmarkSuiteExample
+import XCTest
+
+final class BenchmarkSuiteTests: XCTestCase {
```

Comment:
Do you think we can sugar this up a bit to minimize the amount of boilerplate required by end-users?

Reply:
I just checked using Xcode-provided Swift toolchain, and even this amount of boilerplate is insufficient for automatic discovery to pick things up on macOS. it ignores allTests, and tries to discover tests itself within the test class.

The only solution I see is to provide a single test that runs all benchmarks on macOS (which is not great but better than nothing).

Reply:
I added another workaround to make sure we also run things on macOS.  Overall the current solution is quite dissatisfying but XCTest is simply not flexible enough here.